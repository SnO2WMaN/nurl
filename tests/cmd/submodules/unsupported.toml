args = ["https://crates.io/crates/nurl", "0.3.0", "--submodules"]
