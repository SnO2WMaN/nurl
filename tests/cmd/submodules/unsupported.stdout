fetchCrate {
  pname = "nurl";
  version = "0.3.0";
  hash = "sha256-B6T4DEhE2Jq3YSL+b//27gRkQlvqhynSMBCGdYD5Gog=";
}